docs/apm/using-the-apm-ui.asciidoc

```diff
@@ -113,44 +151,43 @@ This can be extremely useful if it's determined that specific users are getting
 * *Tags* - Useful if you want to start correlating transactions with log files or metrics from Metricbeat.
 * *Custom* - You can configure your agent to add custom contextual information on transactions.
 
-TIP: All of this data is just documents in Elasticsearch.
-This means we can click "Actions - View sample document" to see the actual Elasticsearch document under the discover tab.
+TIP: All of this data is stored in documents in Elasticsearch.
+This means you can select "Actions - View sample document" to see the actual Elasticsearch document under the discover tab.
 
 [[spans]]
-==== Spans
+=== Spans
```

Comment:
Moved Spans to their own page. I think it makes more sense then nesting under transactions.  🦅